opentracing/OpenTracing.hs

```diff
@@ -14,11 +14,17 @@ module OpenTracing
     , traced
     , traced'
 
+    , traced_
+    , traced__
+    , traced'_
+    , traced'__
```

Comment:
these are some aggressive synonyms :)

Reply:
Haha, I figured the RHSes would get old very quickly ;)